src/com/dropbox/plugins/mypy_plugin/MypyConfigLoader.java

```diff
@@ -0,0 +1,42 @@
+package com.dropbox.plugins.mypy_plugin;
+
+import com.dropbox.plugins.mypy_plugin.model.MypyConfig;
+import com.intellij.openapi.project.Project;
+import org.ini4j.Ini;
+
+import java.io.File;
+import java.io.IOException;
+
+public class MypyConfigLoader {
+    final static String DEFAULT_MYPY_COMMAND = "dmypy start -- --follow-imports=error ; dmypy check .";
+    final static String DEFAULT_MYPY_PATH_SUFFIX = "";
+
+    public static MypyConfig findMypyConfig(Project project) {
+        MypyConfigService configService = MypyConfigService.getInstance(project);
+        if (configService != null && configService.getState() != null) {
+            return configService.getState();
+        }
+        MypyConfig iniConfig = loadConfigFromIni(project);
+        if (iniConfig != null) {
+            return iniConfig;
+        }
+        return new MypyConfig(DEFAULT_MYPY_COMMAND, DEFAULT_MYPY_PATH_SUFFIX);
```

Comment:
I would document this precedence order somewhere. I remember there was a basic help window, maybe mention it there, also maybe it makes sense to mention it in `README` (although I am not sure many people outside Dropbox will use this feature, maybe only another companies that that use both mypy and PyCharm).